specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/stable/2025-04-01/ResourceHealth.json

```diff
@@ -0,0 +1,2330 @@
+    },
+    "event": {
+      "type": "object",
+      "description": "Service health event",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "type": "object",
+          "description": "Properties of event.",
+          "properties": {
+            "eventType": {
+              "type": "string",
+              "description": "Type of event.",
+              "enum": [
+                "ServiceIssue",
+                "PlannedMaintenance",
+                "HealthAdvisory",
+                "RCA",
+                "EmergingIssues",
+                "SecurityAdvisory"
+              ],
+              "x-ms-enum": {
+                "name": "EventTypeValues",
+                "modelAsString": true
+              }
+            },
+            "eventSource": {
+              "type": "string",
+              "description": "Source of event.",
+              "enum": [
+                "ResourceHealth",
+                "ServiceHealth"
+              ],
+              "x-ms-enum": {
+                "name": "EventSourceValues",
+                "modelAsString": true
+              }
+            },
+            "status": {
+              "type": "string",
+              "description": "Current status of event.",
+              "enum": [
+                "Active",
+                "Resolved"
+              ],
+              "x-ms-enum": {
+                "name": "EventStatusValues",
+                "modelAsString": true
+              }
+            },
+            "title": {
+              "type": "string",
+              "description": "Title text of event."
+            },
+            "summary": {
+              "type": "string",
+              "description": "Summary text of event. Use fetchEventDetails endpoint to get summary of sensitive events."
+            },
+            "header": {
+              "type": "string",
+              "description": "Header text of event."
+            },
+            "level": {
+              "type": "string",
+              "description": "Level of insight.",
+              "enum": [
+                "Critical",
+                "Warning"
+              ],
+              "x-ms-enum": {
+                "name": "LevelValues",
+                "modelAsString": true
+              }
+            },
+            "eventLevel": {
+              "type": "string",
+              "description": "Level of event.",
+              "enum": [
+                "Critical",
+                "Error",
+                "Warning",
+                "Informational"
+              ],
+              "x-ms-enum": {
+                "name": "EventLevelValues",
+                "modelAsString": true
+              }
+            },
+            "eventSensitive": {
+              "type": "boolean",
+              "description": "If true the event may contains Sensitive data. Use the /fetchEventDetails endpoint to fetch sensitive data"
+            },
```

Comment:
ok first IF this is staying a bool, then it should be called something like 'isEventSensitive' or 'hasSensitiveData'.

But if its NOT staying a bool, then it could perhaps instead be a more useful info, such as: level of sensitivity; or a link to where to fetch the details. 
And then it could have a different name suitable to that purpose.

Reply:
Also if this is just a bool or enum -- then I think this description would be a really good place to give a quick rundown of how to call the /fetchEventDetails endpoint - what what parameters to pass, etc.

Reply:
updated name of property as suggested and linked to documentation for sensitive events